BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-2.0.0-servicing-02012-01
\ No newline at end of file
+2.0.0-servicing-02104-04
```

Comment:
This doesn't work for me locally. In init-tools.log:

```
ERROR: An error occured when running: '"C:\git\coreclr\Tools\dotnetcli\dotnet.exe" restore "C:\git\coreclr\packages\microsoft.dotnet.buildtools\2.0.0-servicing-02104-04\lib\\tool-runtime\project.json"'. Please check above for more details.
```

Pretty sure it's because this version doesn't include https://github.com/dotnet/buildtools/pull/1744. (And CLI 2.0.0 `--version` returns **1**5.5.0-preview-007044.) Updating to BuildTools 2.0.0-servicing-02201-03 (latest) works for me.

Reply:
Updating. Thanks Davis.